Dockerfile

```diff
@@ -27,10 +28,10 @@ RUN apt-get update && apt-get install -y \
 	qtcreator
 
 COPY . /root/mumble
-WORKDIR /root/mumble
+WORKDIR /root/mumble/release
 
-RUN qmake -recursive main.pro CONFIG+="no-client grpc"
-RUN make release
+RUN cmake -Dclient=OFF -Dgrpc=ON ..
+RUN make -j $(nproc)
```

Comment:
I would use Ninja instead.

Reply:
Unnecessary. Ninja isn't faster and is only a dependency that has to be installed in order for the Dockerfile to work. Make on the other hand is part of every Linux distro (that is capable of compiling in the first place)

Reply:
I am in favor for whichever provides the least effort in terms of building the image. As far as Ninja being faster or not, it really depends on the size of the project. I have seen faster builds with it when building both client and server. If the difference is negligible when building just the server, then `make` should be fine.

Reply:
For a fresh build there seems to be almost no difference between make and Ninja: https://david.rothlis.net/ninja-benchmark/
@ZeroAbility are you perhaps referring to Ninja being faster than the default MSBuild on Windows?

Reply:
> are you perhaps referring to Ninja being faster than the default MSBuild on Windows?

Probably, NMake. If there isn't a difference then I would also suggest make.